maker/src/routes.rs

```diff
@@ -62,3 +65,17 @@ pub async fn post_force_close_channel(remote_node_id: String) -> Result<(), Http
 
     Ok(())
 }
+
+#[rocket::post("/open_channel", data = "<request>")]
```

Comment:
nit: it would be nice if we could align the API of opening and closing of the channel, e.g.
- `/api/channel/close/<id>`
- `/api/channel/open/<body>`